plugins/mcreator-localization/lang/texts.properties

```diff
@@ -2807,10 +2807,17 @@ dialog.new_workspace.forge.modid=<html>Mod ID / namespace:<br><small>Used for mo
 dialog.new_workspace.forge.generator=<html>Minecraft version (generator):<br><small>Target Java Edition version
 dialog.new_workspace.forge.package=<html>Mod Java package name:<br><small>Used for root of code generator
 dialog.new_workspace.forge.folder=Workspace folder:      
-dialog.new_workspace.forge.notice1=<html><font color=''#aaaaaa'' size=2>At this time only Minecraft Forge mods offer full feature coverage of MCreator.
-dialog.new_workspace.forge.notice2=<html><font color=''#aaaaaa'' size=2>Minecraft Forge mods are Java based mods that work on Minecraft Java Edition and require\
+dialog.new_workspace.forge.notice=<html><font color=''#aaaaaa'' size=2>Minecraft Forge mods are Java based mods that work on Minecraft Java Edition and require\
   <br>Minecraft Forge to be installed on clients (and the server) in order for them to work.
-dialog.new_workspace.forge.toogle=\ Minecraft Forge mod
+dialog.new_workspace.forge.toggle=\ Minecraft Forge mod
+dialog.new_workspace.neoforge.display_name=<html>Mod display name:<br><small>This name is shown in-game
+dialog.new_workspace.neoforge.modid=<html>Mod ID / namespace:<br><small>Used for mod identification
+dialog.new_workspace.neoforge.generator=<html>Minecraft version (generator):<br><small>Target Java Edition version
+dialog.new_workspace.neoforge.package=<html>Mod Java package name:<br><small>Used for root of code generator
+dialog.new_workspace.neoforge.folder=Workspace folder:      
```

Comment:
Is there a reason for creating multiple keys/string texts for all the different API panels while it's the same thing? It's the same thing for all of them, so it would make sense and it would simplify the task of translators (as well as make the file a bit smaller and more organized)